recipes/pytinyrenderer/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "pytinyrenderer" %}
+{% set version = "0.0.13" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  # Original git repo does not have tags, but this commit is the one corresponding to the 0.0.13 PyPI release
```

Comment:
I assume you want to use a git tag instead of PYPI sdist because you need to patch the build system?

Reply:
Exactly, producing patches again GitHub repos is much easier.